demos/ios/MASVS-AUTH/MASTG-DEMO-0042/MASTG-DEMO-0042.md

```diff
@@ -4,6 +4,30 @@ title: Runtime Use of LAContext.evaluatePolicy with Frida
 id: MASTG-DEMO-0042
 code: [swift]
 test: MASTG-TEST-0267
-status: draft
-note: This demo shows how to detect the use of `LAContext.evaluatePolicy` at runtime using Frida.
 ---
+
+### Sample
+
+This demo uses the same sample as @MASTG-DEMO-0041.
+
+{{ ../MASTG-DEMO-0041/MastgTest.swift }}
+
+### Steps
+
+1. Install the app on a device (@MASTG-TECH-0056)
+2. Make sure you have @MASTG-TOOL-0039 installed on your machine and the frida-server running on the device
+3. Run `run.sh` to spawn your app with Frida
+4. Click the **Start** button
+5. Stop the script by pressing `Ctrl+C`
+
+{{ run.sh # script.js }}
+
+### Observation
+
+{{ output.txt }}
+
+The output reveals the use of `LAContext.evaluatePolicy(0x1, ...)` in the app. Policy `0x1` is [`.deviceOwnerAuthenticationWithBiometrics`](https://developer.apple.com/documentation/localauthentication/lapolicy/deviceownerauthenticationwithbiometrics).
+
+### Evaluation
+
+The test passes because the output shows the runtime use of `LAcontext.evaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, ...)` which verifies whether the app uses an insecure API for biometric authentication.
```

Comment:
This test MUST FAIL because we can only see the `evaluatePolicy` call but no `SecAccessControlCreateWithFlags`

This needs to be aligned with the evaluation of MASTG-TEST-0266:

> The test fails if for each sensitive data resource worth protecting:
> 
> - `LAContext.evaluatePolicy(...)` is used explicitly.
> - There are no calls to `SecAccessControlCreateWithFlags`.

See the suggestion to update run.sh to load an additional frida script. If that works consider updating the output if necessary. The following must be true:

```suggestion
The test fails because the output only shows calls to biometric verification with LocalAuthentication API and no calls to any Keychain APIs requiring user presence.
```